### PR TITLE
release/14.10.0

### LINKED ISSUE #1132: Update to the latest NuGet version of the dependencies

### PR DESCRIPTION
**Pull request by ghost**
fixes #1132  
New Feature
1) Updated all nuget dependencies to their latest version except for ServiceFabric
2) Replaced Pivotal.Discovery.ClientCore with Steeltoe.Discovery.ClientCore as the Pivotal version is deprecated.
3) Changed the port number in a test from 8001 to 86 as 8001 is a port that is often in used on many dev machines.
4) Updated the docs to reflect the change to only support .netcoreapp3.1

### CODE REVIEW

File: src/Ocelot.Administration/Ocelot.Administration.csproj [hunk 1]
```diff
@@ -27,11 +27,11 @@
     <ProjectReference Include="..\Ocelot\Ocelot.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.66">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.164">
```
Comment (jmezach) [inline src/Ocelot.Administration/Ocelot.Administration.csproj]: Still no stable version available of this package?

File: test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs [hunk 2]
```diff
@@ -321,7 +321,7 @@ private void GivenTheBaseUrl(string baseUrl)
 
         private void WhenIGet(string key)
         {
-            _result = _configuration.GetValue("BaseUrl", "");
+            _result = _configuration.GetValue(key, "");
```
Comment (jmezach) [inline test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs]: Was this test broken? If so, why didn't we see this sooner? If not, what changed in the implementation that required this change?

File: test/Ocelot.UnitTests/Kubernetes/KubeServiceDiscoveryProviderTests.cs [hunk 3]
```diff
@@ -38,7 +38,7 @@ public KubeServiceDiscoveryProviderTests()
         {
             _serviceName = "test";
             _namespaces = "dev";
-            _port = 8001;
+            _port = 86;
```
Comment (jmezach) [inline test/Ocelot.UnitTests/Kubernetes/KubeServiceDiscoveryProviderTests.cs]: Was this an intentional change? Or this a machine specific setting somehow?

**Review by jmezach — APPROVED** [review]

File: src/Ocelot.Administration/Ocelot.Administration.csproj (hunk 1 above)
Comment (pedramrezaei) [reply src/Ocelot.Administration/Ocelot.Administration.csproj]: Unfortunately not.

File: test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs (hunk 2 above)
Comment (pedramrezaei) [reply test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs]: It was not broken. The caller always set the key to “BaseUrl” :)

File: test/Ocelot.UnitTests/Kubernetes/KubeServiceDiscoveryProviderTests.cs (hunk 3 above)
Comment (pedramrezaei) [reply test/Ocelot.UnitTests/Kubernetes/KubeServiceDiscoveryProviderTests.cs]: Yes. Intentional. Please see the PR description that has the reasoning.

**Comment by TomPallister** [comment]
@pedramrezaei thanks for the PR, please merge master into your Fork and the build process should now work.

**Comment by pedramrezaei** [comment]
@TomPallister thanks for fixing the issue. This PR is ready for your review. Thanks

**Review by TomPallister — APPROVED** [review]

**Comment by TomPallister** [comment]
@pedramrezaei sorry I have been away for a while and have not merged this. It is the holidays in the UK now so I will do it this weekend.

Thanks for your help with Ocelot.

**Pull request merged**